apps/desks.py

```diff
@@ -537,6 +540,24 @@ def _find_other_sluglines(self, family_id, slugline, versioncreated, desk_id):
         return (False, older_sluglines)
 
 
+class OverviewDesksResource(Resource):
+    url = r'desks/<regex("([a-f0-9]{24})|all"):desk_id>/overview'
+    resource_title = "desk_overview"
+    resource_methods = ['GET']
+
+
+class OverviewDesksService(BaseService):
+
+    def on_fetched(self, doc):
+        desk_id = request.view_args['desk_id']
+        desk_match = {"$ne": None} if desk_id == "all" else ObjectId(desk_id)
+        aggregation = app.data.pymongo().db.archive.aggregate([
```

Comment:
can you use a [timer](https://github.com/superdesk/superdesk-core/blob/develop/superdesk/timer.py#L11) here?
I wonder how much time such aggregation will take

Reply:
Why use Mongo instead of Elasticsearch for aggregations? Elasticsearch is much more equipped for this form of query

Reply:
@petrjasek you want that I log the query time for each query? Because on my local instance it won't tell much, I have just a few items it will be really quick. But logging the query time can be interesting on bigger instances.

@MarkLark86 right ES may be more appropriate, I'll update it.

Reply:
yep I would like to check it later on some bigger instances

Reply:
@MarkLark86 @petrjasek I've updated the PR with timer and it now uses ES

I've measured aggregation with Mongo and ES for comparison, and it seems that on my small local instance, Mongo is a lot faster:

```
overview aggregation mongo '5ef4b152049b9aa6829e8bab': 1.328ms
overview aggregation ES '5ef4b152049b9aa6829e8bab': 7.539ms
overview aggregation mongo 'all': 1.546ms
overview aggregation ES 'all': 6.509ms
```
So I'm not if the move to ES is a good one, we may need to check on bigger instances.

Reply:
Elasticsearch will start to show it's benefits with large data sets. I have witnessed aggregations with the analytics module on 1 years worth of data at AAP complete in milliseconds. Mongo is good at storage, and Elastic is good for searching and aggregations. If Elasticsearch is enabled for a resource, we should always use it

Reply:
@MarkLark86 alright makes sense. Thanks for the feedback, I stay with ES then.